triton-client/src/components/ProjectDetail.tsx

```diff
@@ -1,35 +1,19 @@
-import { useParams } from "react-router-dom"
-import { useAppDispatch, useAppSelector } from "../store/hooks"
+/* tslint:disable */ 
```

Comment:
I disabled tslint, and I saw no errors.

Reply:
ahhh ok, i wondered what happened on the branch, i rebased with your changes and added more quick fixes